src/main/resources/application.yml

```
@@ -0,0 +1,18 @@
+spring:
+  jpa:
+    hibernate:
+      ddl-auto: update
+    properties:
+      hibernate:
+#        show_sql: true
+        format_sql: true
+
+  profiles:
+    active: db
+
+logging.level:
+  org.hibernate.SQL: debug
+# org.hibernate.type: trace
+
+# url: "http://localhost:8080/"
+url: "http://http://15.164.15.13:8080/"
```

이거 아닌가요,....? 그래서인지 배포된 사이트 들어가서 해보니 바로가기 버튼이 정상 작동하지 않더라구요

```suggestion
url: "http://15.164.15.13:8080/"
```

---

아니 이런 실수를,,,! url 수정하는 과정에서 주소가 잘못되었네요... ㅠㅠ 수정했습니다 감사해요!